content/articles/visualizing-repositories-using-pygal/index.md

```diff
@@ -0,0 +1,371 @@
+Pygal allows us to create a range of graphs and charts. You'll learn how to construct a visualisation that shows the comparative popularity of Python works on GitHub in this tutorial. We'll create an interactive bar chart, where the height of each bar will represent the sum of stars earned by the project. By clicking a bar, you'll be sent to the project's GitHub page.
+
+### Prerequisites
+
+You'll need some Python skills to follow along with this tutorial. You must be in a position to work with web APIs in Python.
+
+### Installing pygal
+
+Consider the following command to install pygal:
+
+```python
+
+$ pip install pygal
+
+```
+
+Visit [this](http://www.pygal.org/en/stable/installing.html) if this is your first time using pip.
+
+### The pygal gallery
+
+Check the chart types page to explore what types of charts Pygal can generate: Select **Documentation**, finally **Chart types** in [pygal](http://www.pygal.org/). You can examine how the visualizations are created by looking at the source code for each case.
+
+### Visualizing repositories
+
+Since we'll want to incorporate more than one repository in our visualisation. Let's create a loop that prints specified information about each of the repositories supplied by the API call so that we can add them all to the visualization:
```

Comment:
```suggestion
Since we'll want to incorporate more than one repository in our visualization. Let's create a loop that prints specified information about each of the repositories supplied by the API call so that we can add them all to the visualization.
To achieve this, run the code below:
```
I assume this is the formal beginning of the article. You can add something to indicate and capture reader's attention to diving into the article.